__tests__/app.test.js

```diff
@@ -187,23 +167,81 @@ describe("/api/articles/:article_id", () => {
         });
     });
   });
-})
+});
 
 //ticket 21
-describe('/api/users', () =>{
-  describe('GET', () => {
-    test('200 - returns array of objects with username property', () => {
-      return request(app).get('/api/users')
+describe("/api/users", () => {
+  describe("GET", () => {
+    test("200 - returns array of objects with username property", () => {
+      return request(app)
+        .get("/api/users")
+        .expect(200)
+        .then((res) => {
+          res.body.usernames.forEach((user) => {
+            expect.objectContaining({
+              username: expect.any(String),
+            });
+            expect.not.objectContaining({
+              name: expect.any(String),
+              avatar_url: expect.any(String),
+            });
+          });
+        });
+    });
+  });
+});
+
+//ticket 15
+describe("/api/articles/:article_id/comments", () => {
+  describe("GET", () => {
+    test("200 - given article id responds with all comments for that article. Each comment should have properties: comment_id, votes, created_at, author, body", () => {
+      return request(app)
+        .get("/api/articles/1/comments")
+        .expect(200)
+        .then((res) => {
+          const comments = res.body.comments;
+          expect(comments.length).toBe(11);
+          comments.forEach((comment) => {
+            expect(comment.article_id).toBe(1);
+            expect(comments).not.toEqual([]);
+            expect.objectContaining({
+              comment_id: expect.any(Number),
+              votes: expect.any(Number),
+              created_at: expect.any(String),
+              author: expect.any(String),
+              body: expect.any(String),
+            });
+          });
+        });
+    });
+    test("200 - article exists but has no comments", () => {
+      return request(app)
+        .get("/api/articles/2/comments")
+        .expect(200)
+        .then((res) => {
+          expect(res.body.comments).toEqual([]);
+        });
+    });
+  });
+});
+
+//ticket 9
+describe('/api/articles', () => {
+  describe('GET', ()=>{
+    test('200 - responds with array of article objects with properties: author, title, article_id, topic, created_at, votes. Sorted by descending date', ()=>{
+      return request(app).get('/api/articles')
       .expect(200)
       .then((res)=>{
-        res.body.usernames.forEach((user)=>{
-          expect.objectContaining({
-            username: expect.any(String)
-          })
-          expect.not.objectContaining({
-            name: expect.any(String),
-            avatar_url: expect.any(String)
-          })
+        const {articles} = res.body
```

Comment:
Why are you destructuring here but not above? Try to be more consistent in the way you write your code